### ISSUE TITLE
Pique: Site titles missing on mobile devices

### ISSUE THREAD

**Issue by KristinaKay** [Automattic/themes#2054]
## Steps to replicate

- Activate Pique theme.
- View on Mobile. 

## Result

![Screenshot](https://d.pr/i/8gIov8)

## Expected

The site title should be visible on mobile devices.

* User report: #7960865-hc
* Follow-up ticket: #2992123-zd

**Comment by JessBoctor** [comment]
Looks like this behavior is intentional to show only the site logo on mobile. It only becomes a problem if the site doesn't have a logo. 

This CSS fixes it: 

```
@media all and (max-width: 767px){
.site-logo-link + .site-title, .site-description {
    display: inherit;
}
}
```

**Comment by KristinaKay** [comment]
Thanks. That's not a good UExp.... Hopefully this will be corrected to work differently.

**Comment by nagpai** [comment]
I agree with @KristinaKay 

#24667111-hc

**Comment by kylemcph** [comment]
Another user also agrees with @KristinaKay. #3458502-zen

I found [a forum thread](https://wordpress.com/forums/topic/pique-theme-mobile-view/) referencing this from three years ago as well.

**Comment by carinapilar** [comment]
I also agree that it's not a good experience. I [got a report on forums](https://wordpress.com/br/forums/topic/titulo-do-site-nao-aparece-na-versao-mobile) too.

**Comment by rosepajaroja** [comment]
I have another one here hc-28905275

It happens both on the site title and tagline, and on tablet and mobile view.

**Comment by sourourn** [comment]
Another report: #18867569-hc

**Comment by gwensmithx** [comment]
This seems to be intentional as mentioned previously but also reported here: #4169065-zd

Related: https://github.com/Automattic/themes/issues/21
